### PR TITLE
Doc: Go-live track

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR adds the `Go-live` track.
🔗 [Preview](https://faststore-site-git-vtex-integration-faststore.vercel.app/docs/go-live/overview)

## How it works?

This PR adds the following changes:

- An overview of the go-live process along with a checklist.
- Sections dedicated to future documentation for each step in this track.

P.s. 1: This PR does not include the complete content for the new sections. The focus is an overview and go-live checklist. The content for future documentation will be added in separate PRs.

P.s. 2: The [file to download the checklist](https://github.com/vtex/faststore/pull/1765/files#diff-629beef3bbe7ac0684f68db056e93af84c4c469a5da8df9328771b38d456ee77R25) is still under construction 🚧

### CODE REVIEW

File: apps/site/pages/docs/go-live.mdx
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:1]: Suggestion: move this to be inside the Go-live section and name it "Overview". That's how we're doing on "Getting Started" section. What do you think?

File: apps/site/pages/docs/go-live.mdx [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+This documentation is currently under development.
+</Callout>
+
+**FastStore** is a solution to develop high-performance frontends for ecommerce websites. However, there are other core features that make up a complete digital commerce experience. At **VTEX**, some of these core features are:
+
+- **Checkout** - receives the shopper's information necessary to process an order.
+- **OrderPlaced** - displays a successful message right after the checkout process is completed.
+- **Login** - handles user login in a store.
+- **MyAccount** - allows customers to manage their orders and personal data (e.g., profile info, password, addresses, and credit cards) on a single page.
+- **VTEX Intelligent Search** -
```
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:20]:
```suggestion
- **VTEX Intelligent Search** - displays previous search results based on current and previous searches.
```

File: apps/site/pages/docs/go-live.mdx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+# Overview
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+**FastStore** is a solution to develop high-performance frontends for ecommerce websites. However, there are other core features that make up a complete digital commerce experience. At **VTEX**, some of these core features are:
+
```
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:15]: What o you think of using a table here?

File: apps/site/pages/docs/go-live.mdx [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+<details>
+<summary>Set up and develop your Storefront.</summary>
+| Management | Reference                                                          |
+| :--------- | :----------------------------------------------------------------- |
+| Agency     | [Setting up the project](/docs/getting-started/settingup-project)  |
+</details>
+
+---
+
+### VTEX Integration
```
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:74]:
```suggestion
## VTEX Integration
```

File: apps/site/pages/docs/go-live.mdx [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+<details>
+<summary>Test your website links to pages being rendered in the secure subdomain, such as the Login, Checkout, My Account, and Order Placed pages.</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | Coming soon!  |
+</details>
+
+---
+
+### Headless Customization
```
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:137]:
```suggestion
## Headless Customization
```

File: apps/site/pages/docs/go-live.mdx [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+<details>
+<summary>Set up the preview flow between the storefront and the Headless CMS</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | [VTEX Headless CMS]()  |
+</details>
+
+---
+
+### SSL Configuration
```
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:150]:
```suggestion
## SSL Configuration
```

File: apps/site/pages/docs/go-live.mdx [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+<details>
+<summary>Check if all pages of your store have the correct SSL certificate configured.</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | Coming soon!  |
+</details>
+
+---
+
+### Testing and performance
```
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:163]:
```suggestion
## Testing and performance
```

File: apps/site/pages/docs/go-live.mdx [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+<summary>If you already have a live store, set up an A/B test to validate which version of your website is the best for your business needs.</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency    | Coming soon!  |
+
+</details>
+
+---
+
+### Analytics
```
Comment (renatamottam) [inline apps/site/pages/docs/go-live.mdx:202]:
```suggestion
## Analytics
```

**Review by renatamottam — COMMENTED** [review]
Amazing job ✨ Left a few suggestions below :)

File: apps/site/pages/docs/go-live.mdx
Comment (Mariana-Caetano) [reply apps/site/pages/docs/go-live.mdx:1]: Great idea!

**Review by renatamottam — APPROVED** [review]

File: apps/site/pages/docs/go-live/overview.mdx [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+<details>
+<summary></summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency and VTEX Support    | Coming soon!  |
+
+</details>
+
+
+<b><CheckboxField label="Setting up A/B testing" id="checkboxfield-default" /></b>
```
Comment (gvc) [inline apps/site/pages/docs/go-live/overview.mdx:197]: The correct name for this should be Traffic Split. A/B Testing is a feature that will be enabled by planner in the future.

**Review by gvc — DISMISSED** [review]
Just the A/B Testing thing

File: apps/site/pages/docs/go-live/overview.mdx [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+| :--------- | :------------ |
+| Agency     | Coming soon!  |
+
+- Run Lighthouse audits on your store's Home Page, Product Listing Page (PLP), Product Details Page (PDP), and custom pages.
+- All pages must have Lighthouse Score equal or more than 60 on mobile devices.
+- Any publication will be automatically blocked by the system until all checks pass.
+</details>
+
+
+<b><CheckboxField label="Sentry" id="checkboxfield-default" /></b>
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:186]: @gvc, still not clear to me how the customers will have access to Sentry. Will they have to request access to it?

Comment (gvc) [reply apps/site/pages/docs/go-live/overview.mdx:186]: This is something that's not yet defined by our team. I'll make a note to look after it this afternoon, but I'd go without this for now

File: apps/site/pages/docs/go-live/overview.mdx [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+**FastStore** is a solution to develop high-performance frontends for ecommerce websites. However, there are other core modules that make up a complete digital commerce experience. At **VTEX**, some of these core modules are:
+
+| Module                      | Description                                                                                                                             |
+| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
+| **Checkout**                | Receives the shopper's information necessary to process an order.                                                                       |
+| **OrderPlaced**             | Displays a successful message right after the checkout process is completed.                                                            |
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:24]:
```suggestion
| **OrderPlaced**             | Handles the processing of customer orders. It displays a successful message right after the checkout process is completed.                                                            |
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+| **Login**                   | Handles user login in a store.                                                                                                          |
+| **MyAccount**               | allows customers to manage their orders and personal data (e.g., profile info, password, addresses, and credit cards) on a single page. |
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:26]:
```suggestion
| **MyAccount**               | Allows customers to manage their orders and personal data (e.g., profile information, password, addresses, and credit cards) on a single page. |
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+
+# Overview
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+**FastStore** is a solution to develop high-performance frontends for ecommerce websites. However, there are other core modules that make up a complete digital commerce experience. At **VTEX**, some of these core modules are:
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:19]:
```suggestion
In a headless architecture, an ecommerce website relies on the seamless collaboration of various modules. FastStore specializes in the frontend aspect, enabling the creation of high-performance user interfaces for ecommerce websites. However, to create a complete digital commerce experience, the integration of multiple core modules is necessary. At **VTEX**, some of these modules are:
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+
+| Module                      | Description                                                                                                                             |
+| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
+| **Checkout**                | Receives the shopper's information necessary to process an order.                                                                       |
+| **OrderPlaced**             | Displays a successful message right after the checkout process is completed.                                                            |
+| **Login**                   | Handles user login in a store.                                                                                                          |
+| **MyAccount**               | allows customers to manage their orders and personal data (e.g., profile info, password, addresses, and credit cards) on a single page. |
+| **VTEX Intelligent Search** | Displays previous search results based on current and previous searches.                                                                |
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:27]:
```suggestion
| **VTEX Intelligent Search** | Provides autocomplete and displays relevant results based on the store catalog.                                                                 |
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+title: '  Overview'
+sidebar_label: 'Go-live'
+---
+
+import { Callout, Tab, Tabs } from 'nextra-theme-docs'
+import { CheckboxField } from '@faststore/ui'
+
+<header>
+
+# Overview
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:11]:
```suggestion
# Go-live overview
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+| **OrderPlaced**             | Displays a successful message right after the checkout process is completed.                                                            |
+| **Login**                   | Handles user login in a store.                                                                                                          |
+| **MyAccount**               | allows customers to manage their orders and personal data (e.g., profile info, password, addresses, and credit cards) on a single page. |
+| **VTEX Intelligent Search** | Displays previous search results based on current and previous searches.                                                                |
+
+Therefore, to provide shoppers with a full checkout and post-purchase experience, you, as a developer, must guarantee the integration between your FastStore project and VTEX solutions before going live.
+
+To ensure that you have set up all configurations for your store to run smoothly.
+copy this [store configuration checklist](/tbd) and check if you have configured each item according to your desired operation.
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:32]: Qual é essa lista?

File: apps/site/pages/docs/go-live/overview.mdx [hunk 16]
```diff
@@ -0,0 +1,224 @@
...
+ 
+
+> ⚠️ The checklist above lists the basic features and configurations required to run a VTEX store. It is your team's responsibility to determine which features to implement in your store and how to configure them to create the desired store experience. Therefore, this list may only cover some of the features your store uses, and you must check all aspects of your store's operation, even if they are not on the suggested list. Ainda assim é importante que verifique todos os aspectos do funcionamento da sua loja, mesmo que não estejam na lista sugerida.
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:35]:
```suggestion
> ⚠️ This checklist outlines the basic features and configurations required to run a VTEX store. Hence, this list may cover only some of the features of your store. To address the specific requirements of your store, your team must carefully assess all aspects of your store's functionality, including requirements not explicitly listed here.
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 17]
```diff
@@ -0,0 +1,224 @@
...
+
+
+## Key go-live steps
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:38]:
```suggestion
## Go-live checklist
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+
+To summarize the **essential** steps you **must** take to go-live with stores developed using FastStore, please keep the following in mind:
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:40]:
```suggestion
To ensure a smooth and successful launch of your FastStore project, please consider the following:
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 19]
```diff
@@ -0,0 +1,224 @@
...
+
+### Storefront Developement
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:42]:
```suggestion
### Storefront developement
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 20]
```diff
@@ -0,0 +1,224 @@
...
+
+
+<b><CheckboxField label="Checking the account edition" id="checkboxfield-default" /></b> 
+
+<details>
+<summary>Ensure that your account edition is set to `vtex.edition-store@5.x` Edition app in your master workspace.</summary>
+
+| Management              | Reference                                                                                                                                                                       |
+| :---------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Agency and VTEX Support |  If your account has another Edition app major installed, [open a support ticket](https://help.vtex.com/en/support) requesting the installation of the vtex.edition-store@5.x.  |
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:52]:
```suggestion
| Agency and VTEX Support |  If your account uses a different Edition App major version, [open a support ticket](https://help.vtex.com/en/support) requesting the installation of the vtex.edition-store@5.x.  |
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 21]
```diff
@@ -0,0 +1,224 @@
...
+</details>
+
+
+
+
+<b><CheckboxField label="Deploying with FastStore Onboarding" id="checkboxfield-default" /></b> 
+
+<details>
+<summary>Deploy your project using FastStore Onboarding app.</summary>
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:61]:
```suggestion
<summary>Deploy your project using the FastStore Onboarding app.</summary>
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 22]
```diff
@@ -0,0 +1,224 @@
...
+<details>
+<summary>Set up and develop your Storefront.</summary>
+| Management | Reference                                                          |
+| :--------- | :----------------------------------------------------------------- |
+| Agency     | [Setting up the project](/docs/getting-started/settingup-project)  |
+</details>
+
+---
+
+### VTEX Integration
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:81]:
```suggestion
### VTEX integration
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 23]
```diff
@@ -0,0 +1,224 @@
...
+<details>
+<summary>Test your website links to pages being rendered in the secure subdomain, such as the Login, Checkout, My Account, and Order Placed pages.</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency     | Coming soon!  |
+</details>
+
+---
+
+### Headless Customization
```
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:144]:
```suggestion
### Headless customization
```

**Review by carolinamenezes — COMMENTED** [review]
`go-live/overview` review

**Review by carolinamenezes — COMMENTED** [review]
`go-live/overview` review

File: apps/site/pages/docs/go-live/overview.mdx (hunk 16 above)
Comment (carolinamenezes) [inline apps/site/pages/docs/go-live/overview.mdx:35]: Toda essa parte pode vir na próxima seção.

(acho que faltou dizer que na seção seguinte será apresentada a checklist)

File: apps/site/pages/docs/go-live/overview.mdx [hunk 24]
```diff
@@ -0,0 +1,224 @@
...
+
+# Overview
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+In a headless architecture, an ecommerce website relies on the seamless collaboration of various modules. FastStore specializes in the frontend aspect, enabling the creation of high-performance user interfaces for ecommerce websites. However, to create a complete digital commerce experience, the integration of multiple core modules is necessary. At **VTEX**, some of these modules are:
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:19]:
```suggestion
In a headless architecture, an ecommerce website relies on the seamless collaboration of various modules. FastStore specializes in the frontend aspect, enabling the development of high-performance user interfaces for ecommerce websites. However, to create a complete digital commerce experience, multiple core modules must be integrated. At **VTEX**, some of these modules are:
```

File: apps/site/pages/docs/go-live/overview.mdx (hunk 15 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/go-live/overview.mdx:32]: Estava pensando em uma ideia sugerida pelo Pedro de utilizar a [checklist disponível na trilha "Go Live" do Help](https://help.vtex.com/en/tracks/go-live-your-store--4Ns5FxIiksmjsdX2yOTduM/1iP90RcJvlrfQhnlxM54wo#before-you-begin) e complementá-la com itens específicos da FastStore. No entanto, considerando a [checklist já existente neste PR](https://faststore-site-git-vtex-integration-faststore.vercel.app/docs/go-live/overview#storefront-developement), pode ser redundante adicionar outra lista 🤔

File: apps/site/pages/docs/go-live/before-you-start.mdx [hunk 25]
```diff
@@ -0,0 +1,33 @@
...
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+Before each production go-live, contact the FastStore team at least 4 weeks before the date and make sure that the following was complete:
+
+## Account Edition
+
+Your account edition is set to vtex.edition-store@5.x Edition app in your master workspace
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/before-you-start.mdx:22]:
```suggestion
Your account edition is set to `vtex.edition-store@5.x Edition` app in your master workspace.
```

File: apps/site/pages/docs/go-live/before-you-start.mdx [hunk 26]
```diff
@@ -0,0 +1,33 @@
...
+> If you do not have the appropriate Edition app installed, open a support ticket requesting the installation of the vtex.edition-store@5.x Edition app in your master workspace
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/before-you-start.mdx:23]:
```suggestion
> If you do not have the appropriate Edition app installed, [open a support ticket](https://help.vtex.com/support) requesting the installation of the `vtex.edition-store@5.x Edition` app in your master workspace
```

File: apps/site/pages/docs/go-live/before-you-start.mdx [hunk 27]
```diff
@@ -0,0 +1,33 @@
...
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+Before each production go-live, contact the FastStore team at least 4 weeks before the date and make sure that the following was complete:
+
+## Account Edition
+
+Your account edition is set to `vtex.edition-store@5.x Edition` app in your master workspace.
+> If you do not have the appropriate Edition app installed, [open a support ticket](https://help.vtex.com/support) requesting the installation of the `vtex.edition-store@5.x Edition` app in your master workspace
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/before-you-start.mdx:23]:
```suggestion

<Callout type="warning" emoji="⚠️">
  If you do not have the appropriate Edition app installed, [open a support ticket](https://help.vtex.com/support) requesting the installation of the `vtex.edition-store@5.x Edition` app in your master workspace.
</Callout>
```

**Comment by Mariana-Caetano** [comment]
> Just the A/B Testing thing

Thanks, Guila and the term is now corrected.

File: apps/site/pages/docs/go-live/overview.mdx [hunk 28]
```diff
@@ -0,0 +1,221 @@
...
+This documentation is currently under development.
+</Callout>
+
+In a headless architecture, an ecommerce website relies on the seamless collaboration of various modules. FastStore specializes in the frontend aspect, enabling the development of high-performance user interfaces for ecommerce websites. However, to create a complete digital commerce experience, multiple core modules must be integrated. At **VTEX**, some of these modules are:
+
+| Module                      | Description                                                                                                                             |
+| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
+| **Checkout**                | Receives the shopper's information necessary to process an order.                                                                       |
+| **OrderPlaced**             | Handles the processing of customer orders. It displays a successful message right after the checkout process is completed.                                                            |
```
Comment (gvc) [inline apps/site/pages/docs/go-live/overview.mdx:24]: Isn't the processing part of the checkout?

Comment (Mariana-Caetano) [reply apps/site/pages/docs/go-live/overview.mdx:24]: Yes, Guila!

File: apps/site/pages/docs/go-live/overview.mdx [hunk 29]
```diff
@@ -0,0 +1,221 @@
...
+
+
+
+<b><CheckboxField label="Setting up Traffic Splitting" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>If you already have a live store, configure traffic splitting to validate which version of your website is the best for your business needs.</summary>
+| Management | Reference  -  |
+| :--------- | :------------ |
+| Agency    | Coming soon!  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:201]:
```suggestion
| VTEX Support    | Coming soon!  |
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 30]
```diff
@@ -0,0 +1,221 @@
...
+| :--------- | :----------------------------------------------------------------- |
+| Agency and VTEX Support    | [Integrating the VTEX Order Placed and My Account](/docs/go-live/5-integrating-the-vtex-order-placed-and-my-account)  |
+</details>
+
+
+<b><CheckboxField label="Checking the pages in the secure domain" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Test your website links to pages being rendered in the secure subdomain, such as the Login, Checkout, My Account, and Order Placed pages.</summary>
+| Management | Reference  -  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:134]:
```suggestion
**Management**: Agency
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 31]
```diff
@@ -0,0 +1,221 @@
...
+| :--------- | :------------ |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:135]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 32]
```diff
@@ -0,0 +1,221 @@
...
+| Agency     | Coming soon!  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:136]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 33]
```diff
@@ -0,0 +1,221 @@
...
+
+---
+
+### SSL Configuration
+
+<b><CheckboxField label="Checking the pages SSL certificate" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Check if all pages of your store have the correct SSL certificate configured.</summary>
+| Management | Reference  -  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:160]:
```suggestion
**Management:** Agency
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 34]
```diff
@@ -0,0 +1,221 @@
...
+| :--------- | :------------ |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:161]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 35]
```diff
@@ -0,0 +1,221 @@
...
+| Agency     | Coming soon!  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:162]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 36]
```diff
@@ -0,0 +1,221 @@
...
+
+---
+
+### Testing and performance
+
+<b><CheckboxField label="Performing Lighthouse audits" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Run Lighthouse audits on your store's pages to guarantee you are going live with great performance</summary>
+| Management | Reference  -  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:173]:
```suggestion
**Management:** Agency
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 37]
```diff
@@ -0,0 +1,221 @@
...
+| :--------- | :------------ |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:174]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 38]
```diff
@@ -0,0 +1,221 @@
...
+| Agency     | Coming soon!  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:175]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 39]
```diff
@@ -0,0 +1,221 @@
...
+</details> */}
+
+
+
+
+<b><CheckboxField label="Setting up Traffic Splitting" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>If you already have a live store, configure traffic splitting to validate which version of your website is the best for your business needs.</summary>
+| Management | Reference  -  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:199]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 40]
```diff
@@ -0,0 +1,221 @@
...
+| :--------- | :------------ |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:200]:
```suggestion
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 41]
```diff
@@ -0,0 +1,213 @@
...
+
+---
+
+### Analytics
+
+<b><CheckboxField label="Managing the Analytics module" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Make sure that all analytics events and data are accurately being send to Google Analytics.</summary>
+| Management | Reference  -  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:205]:
```suggestion
| Management | Reference |
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 42]
```diff
@@ -0,0 +1,213 @@
...
+</details> */}
+
+
+
+
+<b><CheckboxField label="Setting up Traffic Splitting" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>If you already have a live store, configure traffic splitting to validate which version of your website is the best for your business needs.</summary>
+| VTEX Support    | Coming soon!  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:193]:
```suggestion
**Management:** VTEX Support
```

File: apps/site/pages/docs/go-live/overview.mdx [hunk 43]
```diff
@@ -0,0 +1,213 @@
...
+
+---
+
+### Headless customization
+
+<b><CheckboxField label="Integrating the storefront with VTEX Headless CMS" id="checkboxfield-default" /></b>       
+
+<details>
+<summary>Set up the preview flow between the storefront and the Headless CMS</summary>
+| Management | Reference  -  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:145]:
```suggestion
| Management | Reference  |
```

**Pull request merged**